tonic/models/vic/grid_params.py

```diff
@@ -322,100 +569,323 @@ def __init__(self, organic_fract=False, spatial_frost=False,
                           'root_depth': 'Root zone thickness (sum of depths is'
                                         ' total depth of root penetration)',
                           'root_fract': 'Fraction of root in the current root '
-                                        'zone.',
-                          'LAI': 'Leaf Area Index, one per month'}
+                                        'zone'}
+        if blowing_snow:
+            self.veg_param['sigma_slope'] = 'Std. deviation of terrain ' \
+                                            + 'slope within veg tile'
+            self.veg_param['lag_one'] = 'Lag one gradient autocorrelation ' \
+                                        + 'of terrain slope'
```

Comment:
When I attempt that, my syntastic add-on to vim complains about an unexpected indent.  But I'll make the change as you suggest regardless...